internal/bminventory/inventory.go

```diff
@@ -4166,7 +4166,17 @@ func formatStaticIPs(staticIpsConfig []*models.StaticIPConfig) string {
 
 	// construct static IPs config string
 	for i, entry := range staticIpsConfig {
-		lines[i] = fmt.Sprintf("%s;%s;%s;%s;%s", entry.Mac, entry.IP, entry.Mask, entry.DNS, entry.Gateway)
+		elements := []string{
```

Comment:
Is everything here is required?

Reply:
No, some are optional but we cannot use a complex formal like JSON here. So because the format is position-based, we need "empty" values. E.g. "mac;;;;;a;b;c;d" means a static IPv4 isn't configured (empty),but a static IPv6 is. @yevgeny-shnaidman do you want to comment?

Reply:
Thanks

Reply:
@filanov as discussed previously, it is easier to keep it in the formatted string. all the fields here are required,  except for ipv4/ipv6 ip fields ( each one can be empty). in the future we will add more fields (vlan, 802.1x, etc)